addons/website/views/snippets/s_timeline.xml

```diff
@@ -79,7 +79,7 @@
         <div data-selector=".s_timeline">
             <we-colorpicker string="Line Color" data-select-style="true" data-css-property="border-color" data-color-prefix="border-" data-apply-to=".s_timeline_line"/>
         </div>
-        <div data-selector=".s_timeline_row">
+        <div data-selector=".s_timeline .s_timeline_row">
```

Comment:
Since you named the rows `s_timeline_list_row` in your snippet, why is this  necessary ?
If it still necessary, I think you should do the same in L75 and L87.

Reply:
Leftover, thanks 👍